source-browser.html

```diff
@@ -10,8 +10,35 @@
     <link rel="stylesheet" href="https://unpkg.com/basscss@7.1.1/css/basscss.min.css" integrity="sha384-/biuPsPEkt10QoeOExIADuWNrpFFXALKPgBhEDkRdKDJdQOPpbNU3uUeFbL/KPJg" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/hint.css/2.5.0/hint.base.min.css" integrity="sha256-ODFgZG+Y3V10lzZu+7J8epM4SdW0w3p8qbVmmShNrtk=" crossorigin="anonymous">
     <style type="text/css">
+      ul:nth-child(1) {
+          list-style-type: none;
+          position: absolute;
+          margin: 0;
```

Comment:
Many of these rules should be replaced with the basscss v7 rules. e.g. `.m0` is `margin: 0`.